packages/components/tag/src/tag.ts

```diff
@@ -104,27 +104,35 @@ export class Tag extends ScopedElementsMixin(LitElement) {
     if (changes.has('disabled')) {
       this.setAttribute('tabindex', this.disabled ? '-1' : '0');
     }
+
+    if (changes.has('removable')) {
+      if (this.removable) {
+        this.setAttribute('aria-description', msg('Press the delete or backspace key to remove this tag'));
```

Comment:
Isn't it necessary to add the information here like it was before in the `aria-label` which tag (`this.label`) is going to be exactly removed?

Reply:
VoiceOver first reads the label of the tag and then this description, so i don't think that is necessary :)

Reply:
ok :)